code/index.html

```diff
@@ -3,12 +3,13 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
+    <title>Guess who?</title>
     <link rel="stylesheet" href="style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet"
     />
+    <link rel="shortcut icon" href="/images/jazebelle.svg" type="image/x-icon">
   </head>
   <body>
     <aside class="question-section">
```

Comment:
Never used this tag, interesting!

Reply:
I actually don't know why its called shortcut icon, the thing I usually use is called a favicon, but it seems to work either way! Nice touch to add a small icon in this way :)